app/assets/javascripts/modules/track-links.js

```diff
@@ -4,10 +4,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 (function (Modules) {
   'use strict'
 
-  Modules.TrackTimelineLinks = function () {
+  Modules.TrackLinks = function () {
```

Comment:
I'm not sure what the `TrackLinks` does that [`TrackClick`](https://github.com/alphagov/static/blob/master/app/assets/javascripts/modules/track-click.js) can not cover. Would you be able to help me understand the need for this script or what can't be achieved by the `track-click` script? I'm thinking we might have been able to avoid [introducing](https://github.com/alphagov/collections/pull/1928) it in the first place.

Reply:
The reason why I can't use track click is that the behaviour of track-click requires you to add the category and action attributes to the link itself. Because we use markdown, we are unable to add attributes to links and this stops us from tracking it.

TrackLinks allows us to apply a module on the wrapper element and track all links within this scope (not just ones with the attributes). I'm going to look into refactoring track-click to support these scenarios but as it is now, it won't work.

Reply:
Makes sense. Thanks for explaining this, Dilwoar! Let's get this in then, and we can consider extending TrackClick to cover this, indeed.

Reply:
Just came across this while looking into a problem with the tracking on these links. Since this PR was raised the `gem-track-click` script has been extended to include the functionality required - you can specify the attributes on a parent element and the tracking label will be set to the URL of the link: https://github.com/alphagov/govuk_publishing_components/blob/master/docs/analytics/track-click.md

I'll look into whether `TrackLinks` is still needed as part of fixing the tracking issue.